tests/handlers/fileserver_test.py

```diff
@@ -0,0 +1,66 @@
+"""Tests for user lab routes."""
```

Comment:
This is a good start for tests. Testing also needed:
- Behavior when the file server already exists
- Reconciliation when the job exits due to a timeout (if we keep that architecture)
- Timeout where the job never starts
- Timeout where the ingress never gets created
- Should start the file server creation first, and then create the ingress, so that you check that the waiting for ingress creation is done properly

Reply:
noted; will fix

Reply:
Will copy to DM-38972